atomic_defi_design/qml/Settings/SettingModal.qml

```diff
@@ -18,9 +18,14 @@ import "../Constants"
 Qaterial.Dialog {
 
     function disconnect() {
-        app.currentWalletName = ""
-        API.app.disconnect()
-        onDisconnect()
+        
+        Qaterial.DialogManager.showDialog({title: qsTr("Logout request"),text: qsTr("Confirm logout request to disconnect!!"),iconSource: Qaterial.Icons.logout,standardButtons: Dialog.Yes | Dialog.Cancel, onAccepted: function(){

```

Comment:
Change title to "Confirm Logout"
Change text to "Are you sure you want to log out?"